### ISSUE TITLE
Tests fail when PERL_DATETIME_DEFAULT_TZ is set on system

### ISSUE THREAD

**Issue by realflash** [houseabsolute/DateTime.pm#128]
Installing latest CPAN release 1.54. Various test failures occur. [Full log](https://gist.github.com/realflash/995f53e45332b38235debc12a23b6919)

I suspect this one is representative of the general problem:
```
Failed test 'Time zones for new DateTime objects should default to floating'
#   at t/47default-time-zone.t line 10.
#          got: 'Europe/London'
#     expected: 'floating'
```
This is a line in my /etc/environment:
`PERL_DATETIME_DEFAULT_TZ="Europe/London"`

I did this so any Perl code running on my system use DateTime would behave appropriately. I presume I am correct in using it this way. 

If I comment out this line and then reattempt the install it completes successfully.

**Comment by n1vux** [comment]
Interesting, thank you.
Looks like _you_ can ignore that error.

Another thing for tests to mock or unset, as tests should not depend on local settings.

**Comment by autarch** [comment]
Fixed in v1.58. Thanks for the bug report!
